python/modem-firmware-1.3+/device_credentials_installer.py

```diff
@@ -84,6 +84,9 @@ def parse_args():
     parser.add_argument("-g", "--gateway",
                         help="Force use of shell commands to enter and exit AT command mode",
                         action='store_true', default=False)
+    parser.add_argument("--shell",
```

Comment:
Rather than add a new `--shell` argument, can you expand the `--cmd-type` argument to accept a third value: `CMD_TYPE_AT_SHELL`?